tests/test_auth.py

```diff
@@ -36,6 +36,7 @@ class Fixtures:
 
```

Comment:
Can we add a test that checks that `aud` is set to `client_id` when `audience` is not specified in the `get_auth` invocation?